### PR TITLE
[ONNX] fix export of embedding with padding_idx

### PR DESCRIPTION
**Pull request by neginraoof**
fix export of embedding with padding_idx

### CODE REVIEW

File: torch/onnx/symbolic_opset9.py [hunk 1]
```diff
@@ -471,8 +471,17 @@ def expand_as(g, self, other):
     return g.op("Expand", self, shape)
 
 
+@parse_args('v', 'v', 'i', 'v', 'v')
 def embedding(g, weight, indices, padding_idx, scale_grad_by_freq, sparse):
-    return g.op("Gather", weight, indices)
+    # if (padding_idx >= 0) {
```
Comment (jiafatom) [inline torch/onnx/symbolic_opset9.py:476]: What does this mean? Seems that for `padding_idx >= 0`, the pytorch statement becomes `embedding.masked_fill_((indices == padding_idx).reshape({-1, 1}), 0);` for embedding operator. If this is the case, maybe it's better to put it in the `if (padding_idx >= 0):` block below.

Comment (BowenBao) [reply torch/onnx/symbolic_opset9.py:476]: I guess this is reference code for pytorch implementation. Might be better if this is clarified in comments.

File: torch/onnx/symbolic_opset9.py [hunk 2]
```diff
@@ -471,8 +471,17 @@ def expand_as(g, self, other):
     return g.op("Expand", self, shape)
 
 
+@parse_args('v', 'v', 'i', 'v', 'v')
 def embedding(g, weight, indices, padding_idx, scale_grad_by_freq, sparse):
```
Comment (spandantiwari) [inline torch/onnx/symbolic_opset9.py:475]: @Derek pointed out that we are not handling the `scale_grad_by_freq` argument, which affects how gradients are scaled in the backward pass for this op. That is relevant for training. What can we do about that? 
At the very least, we can check if the export is in training mode and `scale_grad_by_freq=True`, and error out or warn. Let's think about that.

Comment (neginraoof) [reply torch/onnx/symbolic_opset9.py:475]: Addressed.

**Review by BowenBao — APPROVED** [review]
LGTM, thanks.
